README.md

```diff
@@ -1 +1,31 @@
-# som_timer_be
\ No newline at end of file
+# som_timer_be
+
+## Endpoints
+
+#### GET  '/api/timers/{timer_id}
+
+```
+#RESPONSE
+{
+    "id": 1,
+    "work_interval": "25:00",
+    "rest_interval": "5:00"
+}
+```
+
+#### PUT  '/api/timers/{timer_id}
+
+```
+#REQUEST BODY
```

Comment:
Great documentation, y'all!! 

I'm curious about how this will work with our current front end functionality, in which users can edit each interval separately! Do we think it makes more sense to use a submit button on timer settings to minimize the number of requests we're making? Or does it make sense to make a network request each time the user updates either field?

Reply:
A submit button will probably be better in regards to long term processing power but doing more requests at this stage will be negligible.